msvc/Makefile.am

```diff
@@ -4,14 +4,8 @@ VCXPROJS = $(wildcard *.vcxproj)
 EXTRA_DIST = \
 	     $(VCXPROJS) \
 	     README \
-	     create-windef.pl \
-	     mono.def \
 	     mono.props \
 	     mono.sln \
 	     monoposixhelper.def \
```

Comment:
What about this .def file?

Reply:
This PR intentionally left that behind since the source was not prepared with __declspec (dllexport) and some of the exported functions was also defined in external headers. This file is not as much updated as the def file for libmono and could be dropped later, if we feel it is worth the effort to fix the source files to get there.